subsys/net/ip/Kconfig

```diff
@@ -373,7 +373,8 @@ config NET_CONTEXT_TIMESTAMP
 	bool "Add timepstamp support to net_context"
 	select NET_PKT_TIMESTAMP
 	help
-	  It is possible to timestamp outgoing packets.
+	  It is possible to timestamp outgoing packets and get information
```

Comment:
I find such descriptions confusing. It's possible to timestamp outgoing packets, good. Then, suddenly, it's possible to get information about net_pkt timings. How "timestamp outgoing packets" and "net_pkt timings" relate? Any reason this doesn't say "It is possible to timestamp outgoing packets and get information about these timestamps" ?